Ui/src/Components/LightingPage/Dialogs/PowerDialog/PowerDialog.tsx

```diff
@@ -67,30 +66,27 @@ const PowerDialog: FC = () => {
   }, []);
 
   const onClickBulbOnePower = () => {
-    setBulbOneState(!bulbOneState);
-    const powerRequest: LightingRequest = {
-      operation: bulbOneState ? "off" : "on",
-    };
-
-    post(config.BULB_1_ENDPOINT + "/request", powerRequest);
+    // setBulbOneState(!bulbOneState);
```

Comment:
RIP @julian-patterson 

We'll have to find a new way to add scenes. I'll make an issue

What we be awesome is user set scenes, so a modal to add a scene under the scene page. Similar to what we did for add your own device, but you set the scene using the devices you have added. Would be insane.